django_slack_oauth/models.py

```diff
@@ -18,8 +15,23 @@ class SlackUser(models.Model):
     def is_slacked(self):
         return self.access_token
 
-    def __unicode__(self):
-        return unicode(self.slacker)
+    def __str__(self):
+        return self.slacker.__str__()
 
     class Meta:
         db_table = 'slack_user'
+
+
+class SlackOAuthRequest(models.Model):
```

Comment:
@izdi, agreed on the version update. It makes things a lot simpler and gives us a chance to implement.